packages/eds-core-react/src/components/Dialog/DialogHeader.tsx

```diff
@@ -8,7 +8,7 @@ const StyledDialogHeader = styled.div(({ theme }) => {
   return css`
     display: flex;
     justify-content: space-betweene;
-    algin-items: center;
+    align-items: center;
```

Comment:
Uhm, suggest making a separate issue for tracebility for this as the above line is also wrong "space-betweene"....

Reply:
Wasn’t `Stylelint` extension  behind this idea?

Reply:
Probably, I am getting the `align-items` as error in my IDE but not the justify-content on, which has the wrong value.

![image](https://user-images.githubusercontent.com/1070981/184843793-9f1e2330-bc5a-4330-966d-2eaabace6646.png)

Reply:
Thats right, `justify-content` haven't been underline.

Reply:
done, new issue #2417

Reply:
Suggest reverting these changes and do the fixes in a separate PR for traceability